config/locales/ua.yml

```diff
@@ -139,7 +118,7 @@ ua:
         role: Роль
         approved: Підтвердженний
         encrypted_password: "Зашифрований пароль"
-        failed_attempts: "Хибних спроб"
+        failed_attempts: "Хибниз спроб"
```

Comment:
This commit 
https://github.com/BohdanChaban/school/pull/38/commits/b7daa8529cc273f657a9ce085866ae5d622b6009
make some strange changes, remove translations. Is it ok?

Reply:
it`s wrong rollback :(